src/EditorFeatures/VisualBasicTest/Structure/MultiLineIfBlockStructureTests.vb

```diff
@@ -16,30 +16,93 @@ Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.UnitTests.Outlining
         Public Async Function TestIfBlock1() As Task
             Const code = "
 Class C
-    Sub M()
-        {|span:If (True) $$
-        End If|}
-    End Sub
+Sub M()
+{|span:If (True) $$
+End If|}
+End Sub
```

Comment:
why was all of this dedented?

Reply:
It made it easier to find my errors when developing this feature.

Reply:
Please restore things.